cross-chain/arbitrum/deploy_l2/01_deploy_arbitrum_tbtc_token.ts

```diff
@@ -0,0 +1,39 @@
+import type { HardhatRuntimeEnvironment } from "hardhat/types"
+import type { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { ethers, getNamedAccounts, helpers } = hre
+  const { deployer } = await getNamedAccounts()
+
+  const [, proxyDeployment] = await helpers.upgrades.deployProxy(
+    "ArbitrumTBTC",
+    {
+      contractName: "L2TBTC",
```

Comment:
Being specific here may help us reduce confusion.

```suggestion
      contractName: "@keep-network/tbtc-v2/contracts/l2/L2TBTC.sol:L2TBTC",
```

Reply:
[d2c5a17](https://github.com/keep-network/tbtc-v2/pull/554/commits/d2c5a174919fabe37252673cc75249341c06e1fb)